### ISSUE TITLE
scaling collected_torrents

### ISSUE THREAD

**Issue by Dmole** [Tribler/tribler#3960]
```
> du -sc ~/.Tribler/collected_torrents 
973M	~/.Tribler/collected_torrents
> strings ~/.Tribler/collected_torrents/*.ldb | grep files | wc -l
19877
```
That is not going to scale well, I recommend only storing torrents for subscribed channels.
Or share the Max Disk space setting from token mining.

**Comment by devos50** [comment]
We recently fixed an issue where torrents were not collected at all, which explains why the size of this file is growing. IIRC, we had some code to remove old torrents from the leveldb store but I'm not sure what happened to that. Also, there used to be a configuration option to limit the total number of torrents in the store (this was 50.000 by default I think).

Storing only torrents for subscribed channels is a good idea. Another solution could be to only store popular torrents since they are more likely to be downloaded by the user. Maybe we could combine this with the popularity overlay that @xoriole made?

**Comment by Dmole** [comment]
Maybe 2 options;
-prioritize [recent, popular, subscribed]
-limit up to [X] MB
and a "clear cache" button

**Comment by Dmole** [comment]
```
rm -r .Tribler/collected_*
```
reduced the size from 1.2 GB to 7MB but cost a lot of ratio/trust.
https://github.com/Tribler/tribler/issues/3657
should not be closed IMO.

**Comment by Dmole** [comment]
.Tribler/collected_* should be moved to $Downloads with credit_mining.

**Comment by devos50** [comment]
Related to #1150

**Comment by Dmole** [comment]
https://github.com/Tribler/tribler/pull/4199

**Comment by ichorid** [comment]
Fixed by #4090
